duck.sh

```diff
@@ -42,17 +51,31 @@ if [[ "${INTERVAL: -1}" == 'm' && "${INTERVAL:0:-1}" -lt 5 ]]; then
   exit 1
 fi
 
+#https://www.duckdns.org/update?domains={YOURVALUE}&token={YOURVALUE}[&ip={YOURVALUE}][&ipv6={YOURVALUE}][&verbose=true][&clear=true]
+endpoint=https://www.duckdns.org/update
+
 #-----------------------------------------------------------------------------------------------------------------------
 
 function ts {
   echo [`date '+%b %d %X'`]
 }
 
 #-----------------------------------------------------------------------------------------------------------------------
+sleep 2
```

Comment:
Do not remember why I wrote it. Cannot make any sense about it now... could be removed?